src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests.cs

```diff
@@ -16309,7 +16309,6 @@ static void Main()
                 }
                 """;
             comp = CreateCompilation(sourceB, references: new[] { refA }, targetFramework: TargetFramework.Net80);
-            // PROTOTYPE: ERR_CollectionBuilderAttributeMethodNotFound error message is stale. It should say "... with a first parameter of type 'ReadOnlySpan<object> callable with no additional arguments, ...".
```

Comment:
>PROTOTYPE [](http://example.com/codeflow?start=15&length=9)

The `ReadOnlySpan<T>` parameter should be last, not first, so the current message seems sufficient.